internal/incrementalcompiler-core/src/main/scala/sbt/internal/inc/MemberRefInvalidator.scala

```diff
@@ -53,65 +53,59 @@ import xsbt.api.APIUtil
  * of regular members then we'll invalidate sources that use those names.
  */
```

Comment:
The whole comment for this class needs to be updated since it describes the strategy as being implemented as T => Set[File]` whereas now it's `String => Set[String]`.